src/renderers/art/ReactART.js

```diff
@@ -93,7 +93,7 @@ function injectAfter(parentNode, referenceNode, node) {
 
 // ContainerMixin for components that can hold ART nodes
 
-const ContainerMixin = assign({}, ReactMultiChild, {
+const ContainerMixin = assign({}, ReactMultiChild.prototype, {
 
```

Comment:
It shouldn't be AFAIK, it's still extending the same methods. They just exist on a class now instead of an object.